app/move/controllers/view/view.locals.js

```diff
@@ -61,11 +65,12 @@ function getViewLocals(req) {
     sectionProgress: personEscortRecord?.meta?.section_progress,
   })
   const personEscortRecordTagList = presenters.frameworkFlagsToTagList(
-    personEscortRecord?.flags
+    personEscortRecord?.flags,
+    moveUrl
```

Comment:
I think you will need to do something similar in `assessmentToTagList` on [line 120](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/1015/files#diff-5df44f9cefd3c5c02a47898f03d1a99a5f469e90f8ea9e34174c7844734cab4bR120).

The assessment answers are still shown as tags for users that don't have the Person Escort Record role enabled.

Reply:
Fixed